spec.html

```diff
@@ -36380,7 +36380,7 @@
     <!-- es6num="B.3.1" -->
     <emu-annex id="sec-__proto__-property-names-in-object-initializers">
       <h1>__proto__ Property Names in Object Initializers</h1>
-      <p>The following Early Error rule is added to those in <emu-xref href="#sec-object-initializer-static-semantics-early-errors"></emu-xref>:</p>
+      <p>The following Early Error rule is added to those in <emu-xref href="#sec-object-initializer-static-semantics-early-errors"></emu-xref>. When |ObjectLiteral| appears in a context where |ObjectAssignmentPattern| is required the Early Error rule is <b>not</b> applied. In addition, it is not applied when initially parsing a |CoverParenthesizedExpressionAndArrowParameterList|.</p>
```

Comment:
The spec mostly just uses `<em>` at the defining occurrence of terms (at least, the ones that don't rate a `<dfn>`).